public/scripts/dependencies/appwrite.js

```diff
@@ -665,11 +665,11 @@
                 }
 
                 if(passwordA) {
```

Comment:
Maybe these variables also should be updated to `password` and `passwordAgain`?

Reply:
👌🏻

Reply:
Fixed